.github/workflows/ci.yml

```diff
@@ -245,21 +252,64 @@ jobs:
         with:
           java-version: 1.8
 
-      - name: Build unscoped PHAR for testing
+      - name: Test building an unscoped PHAR
         run: ant unscoped-phar-snapshot
 
-      - name: Run regular tests with unscoped PHAR
-        run: ./build/artifacts/phpunit-snapshot.phar
-
-      - name: Build scoped PHAR for testing
+      - name: Test building a scoped PHAR
         run: ant phar-snapshot
 
-      - name: Run PHAR-specific tests with scoped PHAR
-        run: ./build/artifacts/phpunit-snapshot.phar --configuration tests/phar/phpunit.xml
-
-      - uses: actions/upload-artifact@v2
+      - name: Upload scoped Phar build on PHP 8.0
+        uses: actions/upload-artifact@v2
         if: ${{ matrix.php-version == 8.0 }}
         with:
           name: phpunit-snapshot-phar
           path: ./build/artifacts/phpunit-snapshot.phar
           retention-days: 7
+
+  test-phar:
+    name: Test PHAR
+
+    runs-on: ubuntu-latest
+
+    needs:
+      - build-phar
+
+    env:
+      PHP_EXTENSIONS: dom, json, libxml, mbstring, pdo_sqlite, soap, xml, xmlwriter
+      PHP_INI_VALUES: assert.exception=1, zend.assertions=1, error_reporting=-1, log_errors_max_len=0, display_errors=On
+
+    strategy:
+      fail-fast: false
```

Comment:
```suggestion
      fail-fast: false

```

Reply:
The extra new line here would be inconsistent when compared to the same directive everywhere else in the script.